coral-common/src/main/java/com/linkedin/coral/common/calcite/sql/SqlCreateTable.java

```diff
@@ -0,0 +1,91 @@
+package com.linkedin.coral.common.calcite.sql;
+
+import com.linkedin.coral.javax.annotation.Nullable;
+import org.apache.calcite.sql.*;
+import org.apache.calcite.sql.parser.SqlParserPos;
+import org.apache.calcite.util.ImmutableNullableList;
+
+import java.util.List;
+import java.util.Objects;
+
+public class SqlCreateTable extends SqlCreate {
+    private final SqlIdentifier name;
+    private final @Nullable SqlNodeList columnList;
+    private @Nullable SqlNode query;
+    private final @Nullable SqlNode tableSerializer;
+    private final @Nullable SqlNodeList tableFileFormat;
+    private final @Nullable SqlCharStringLiteral tableRowFormat;
+
+    private static final SqlOperator OPERATOR =
+            new SqlSpecialOperator("CREATE TABLE", SqlKind.CREATE_TABLE);
+
+    /** Creates a SqlCreateTable. */
+    public SqlCreateTable(SqlParserPos pos, boolean replace, boolean ifNotExists,
+                          SqlIdentifier name, @Nullable SqlNodeList columnList, @Nullable SqlNode query, SqlNode tableSerializer, SqlNodeList tableFileFormat, SqlCharStringLiteral tableRowFormat) {
+        super(OPERATOR, pos, replace, ifNotExists);
+        this.name = Objects.requireNonNull(name, "name");
+        this.columnList = columnList; // may be null, like in case of ctas
+        this.query = query; // for "CREATE TABLE ... AS query"; may be null
+        this.tableSerializer = tableSerializer;
+        this.tableFileFormat = tableFileFormat;
+        this.tableRowFormat = tableRowFormat;
+    }
+
+    @SuppressWarnings("nullness")
+    @Override public List<SqlNode> getOperandList() {
+        return ImmutableNullableList.of(name, columnList, query);
+    }
+
+    @Override public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
+        writer.keyword("CREATE");
```

Comment:
nit: line 40 & 41 can be combined

Reply:
better to keep it separate, later we will have support for temporary and external table, which will be prefix to `TABLE` keyword